app/subscriber/src/components/date-filter/DateFilter.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  IContentListAdvancedFilter,
+  IContentListFilter,
+} from 'features/content/list-view/interfaces';
+import React from 'react';
+import ReactDatePicker from 'react-datepicker';
+import { FaAngleLeft, FaAngleRight, FaCalendarAlt } from 'react-icons/fa';
+import { useContent } from 'store/hooks';
+import { IContentModel, Page } from 'tno-core';
+
+import * as styled from './styled';
+
+export interface IDateFilterProps {
+  fetch: (
+    filter: IContentListFilter & Partial<IContentListAdvancedFilter>,
+  ) => Promise<Page<IContentModel>>;
+}
+
+/** Custom datefilter for the subscriber home page. Control the calendar state with custom button, custom styling also applied. Also allows user to navigate a day at a time via arrow buttons. */
+export const DateFilter: React.FC<IDateFilterProps> = ({ fetch }) => {
```

Comment:
I may be blind but I don't see you using this property in the component?  Remove it if you aren't.